cmake/scripts/ROOTConfig.cmake.in

```diff
@@ -90,6 +90,15 @@ if (NOT ROOT_builtin_nlohmannjson_FOUND)
   include(CMakeFindDependencyMacro)
   find_dependency(nlohmann_json @nlohmann_json_VERSION@)
 endif()
+if(ROOT_minuit2_mpi_FOUND)
+  include(CMakeFindDependencyMacro)
+  find_dependency(MPI)
+endif()
+if(ROOT_minuit2_omp_FOUND)
+  include(CMakeFindDependencyMacro)
```

Comment:
I guesss this could be included unconditionally to simplify things a bit, what do you think?

Reply:
Well, depends... Is it possible to have only one of those two options set?

Reply:
you're right :-)